css/properties/min-width.json

```diff
@@ -304,11 +304,12 @@
               },
               "safari": [
                 {
-                  "version_added": true
+                  "prefix": "-webkit-",
+                  "version_added": "6.1"
                 },
                 {
-                  "alternative_name": "-webkit-fill-available",
-                  "version_added": "6.1"
+                  "alternative_name": "intrinsic",
```

Comment:
Intrinsic isn't a real value, is it? See also https://github.com/mdn/browser-compat-data/issues/4256

Reply:
That's what's currently in `max-width`'s `fit-content`.  I'm not certain on the details of the validity of `intrinsic`, perhaps something that should be dug into more?

Reply:
woah, you're right this was in fact a keyword in Safari. 
"Older webkit browsers also support the unofficial intrinsic value which acts the same as max-content." says https://caniuse.com/#feat=intrinsic-width

Not sure it is correct to also have this as an alternative to fit-content. Maybe it is?

Reply:
Looking around in this same file, I found the exact same data applied to `max-content` rather than `fit-content`, which...doesn't make too much sense of why it's in `max-width` as `fit-content` as well.

This is another change that should probably be cherry-picked and dug into more.  I'll go ahead and revert changes in this PR, and submit a new one shortly!